src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -119,7 +119,34 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
 
   --cec8e8123c05ba25--
   ```
+  ```client-react-native
+  import { Client, Storage, ID, Permission } from 'react-native-appwrite';
 
+  const client = new Client()
+      .setEndpoint('https://cloud.appwrite.io/v1') // Your API Endpoint
+      .setProject('<PROJECT_ID>');                 // Your project ID
+
+  const storage = new Storage(client);
+
+  const promise = await storage.createFile(
+      '[BUCKET_ID]',
```

Comment:
@choir27 We use `<>` angle brackets universally for placeholders now. I started moving toward this but didn't finish. Make sure you stay consistent!